R/activate.R

```diff
@@ -156,22 +156,26 @@ renv_activate_prompt <- function(action, library, prompt, project) {
   if (!ask)
     return(FALSE)
 
-  fmt <- lines(
-    "",
-    "This project has not yet been activated.",
-    "Activating this project will ensure the project library is used when %s() is called.",
-    "Please see `?renv::activate` for more details.",
-    ""
-  )
-  writef(fmt, action)
+  renv_activate_prompt_impl(action, project)
 
-  fmt <- "Would you like to activate this project before %s() is called?"
-  question <- sprintf(fmt, action)
-  response <- ask(question, default = TRUE)
-  if (!response)
-    return(FALSE)
 
-  activate(project = project)
-  return(TRUE)
+}
 
+renv_activate_prompt_impl <- function(action, project = NULL) {
+  title <- c(
+    sprintf("It looks like you've called renv::%s() in a non-renv project.", action),
```

Comment:
Hmmmm, I guess I was a bit confused by the focus of the function on "activate", rather than "load". I wonder if it's worth distinguishing between the two cases (i.e. non-renv vs not-loaded renv) or just writing something generic. Previously we had:

> This project has not yet been activated.

So maybe 

> It looks like you've called renv::%s() in a project that hasn't been activated yet

?

Reply:
Yeah, I like that.

For extra context, the main reason why I imagined this prompt being necessary: imagine you share an `renv` project with someone else, and so you just share the project + the lockfile. Your collaborator starts working and runs `renv::restore()`.

This way, we can see they probably intended to load the project first so they could use a project library, and also that we can "activate" the project so the project library is loaded by default in the future.

Reply:
Ah, that makes sense.